### PR TITLE
Add new information to the mails sent to subscribers

### PR DESCRIPTION
**Pull request by pratid**
#### Reviewers

@flopezag @jframos 
#### Description

This solves issue #112:
- Send the new region status (Sanity Status) as part of each mail notification.
- Add a link to the report at FIHealth site in such notifications.
#### Testing

Verified.

### CODE REVIEW

File: dashboard/lib/routes/subscribe.js [hunk 1]
```diff
@@ -149,18 +149,20 @@ function searchSubscription(user, regions, callback) {
...
+        regionStatus = region.status;
 
+    logger.info({op: 'subscribe#notify'}, 'notify to region: ' + regionName);
 
     var payloadString = 'name_from= fi-health sanity&';
         payloadString += 'email_from=' + config.mailman.emailFrom + '&';
-        payloadString += 'subject=Status changed for region ' + region + '&';
-        payloadString += 'body=Status changed for region ' + region;
+        payloadString += 'subject=Status changed for region ' + regionName + '&';
+        payloadString += 'body=Status changed to ' + regionStatus + ' for region ' + regionName;
```
Comment (flopezag) [inline dashboard/lib/routes/subscribe.js:167]: why you do not add a link to the fi-health?

Comment (pratid) [inline dashboard/lib/routes/subscribe.js:167]: not possible with current Dashboard implementation

Comment (jframos) [inline dashboard/lib/routes/subscribe.js]: Some questions that I think are interesting to be considered:
- Why don't we use **HTML** to put some style to the 'status'?
- Why don't we send **timestamp** and **elapsed time** information?

File: dashboard/lib/routes/subscribe.js [hunk 2]
```diff
@@ -149,18 +149,21 @@ function searchSubscription(user, regions, callback) {
...
 
+    logger.info({op: 'subscribe#notify'}, 'notify to region: ' + regionName);
 
     var payloadString = 'name_from= fi-health sanity&';
         payloadString += 'email_from=' + config.mailman.emailFrom + '&';
-        payloadString += 'subject=Status changed for region ' + region + '&';
-        payloadString += 'body=Status changed for region ' + region;
+        payloadString += 'subject=Status changed for region ' + regionName + '&';
+        payloadString += 'body=Status changed to ' + regionStatus + ' for region ' + regionName +
+                         ' (visit ' + config.fiHealthUrl + ' for details)';
```
Comment (jframos) [inline dashboard/lib/routes/subscribe.js]: We are already sending the URL in the body of the message. It is defined in the Mailman template we use.

Current body: 

```
 From: spain2@fi-health.lab.fiware.org
 To:   FiHealth mailing list for region spain2 <spain2@fi-health.lab.fiware.org>

    Status changed for region Spain2 
    _______________________________________________
    Fi-Health Sanity Checks 
    http://fi-health.lab.fiware.org/
```

**Comment by jframos** [comment]
LGTM

**Comment by flopezag** [comment]
LGTM

**Pull request merged**